assignments/4. operators/loops.md

```diff
@@ -14,16 +14,48 @@ const ACCESSORY_PRICE = 9.99;
 var bank_balance = 303.91;
 var amount = 0;
 // your code goes here
+for (; PHONE_PRICE <= bank_balance; bank_balance - PHONE_PRICE) {
+    bank_balance = bank_balance - PHONE_PRICE;
+    amount = amount + PHONE_PRICE;
+
+    for (; ACCESSORY_PRICE <= SPENDING_THRESHOLD; bank_balance - ACCESSORY_PRICE) {
+        bank_balance = bank_balance - ACCESSORY_PRICE;
+        amount = amount + ACCESSORY_PRICE;
+        break;
+    }
+}
+// Calculate tax function
+let total = 0;
+let calculateTax = (amount) => {
+    total = amount * TAX_RATE;
+    return total;
+}
+
+// Formate amount function
+let formatAmount = (amount) => `$${amount.toFixed(2)}`;
+
+// Check amount against account bal
+let permission = (total <= bank_balance) ? 'Yes, you can buy it' : 'You don\'t have enough money in your account'; 
 ```
  ⛑ Answer of the above will `$334.76`.
 
 2. 🎖 Write a for loop that will iterate from 0 to 20. For each iteration, it will check if the current number is even or odd, and report that to the screen using `alert` (e.g. "2 is even").
 ```js
 // your code goes here
-```
+for ( let num = 1; num <= 20; ++num ) {
+    if ( num % 2 == 0) {
+        alert(`${num} is even`);
+    }
+}
```

Comment:
You also need to `alert` if the number is odd. (it will check if the current number is even or odd)